### DISCUSSION TITLE
FR: Automatically assign 4he the recipe source as a tag

### DISCUSSION THREAD

**Discussion by PeterJT** [mealie-recipes/mealie#2405] [Feature Request]
When creating a new recipe by scraping a website, add the website's URL as a tag.

Possibly have this as an installation / admin settable option; not one that needs to be specified on recipe import.

**Comment by michael-genson** [comment]
You may already know this, but when you create a recipe via URL, you can scroll to the bottom of the recipe to find the "Original URL" button:
![image](https://github.com/hay-kot/mealie/assets/71845777/dc44f0fd-f93c-4751-9445-92b9884bd9c6)


Not exactly what you're requesting, but figured I'd mention it just in case you hadn't seen it

**Reply by PeterJT** [reply]
Fully aware of that. It would just be a convenient (and quick) way to select all recipes from a given source

**Comment by Charlotte-br560** [comment]
Automatically tagging recipes with their source URL sounds like a handy feature. Having it configurable by admins would add flexibility. Additionally, you might find [Crawlbase](https://crawlbase.com/) useful for web scraping tasks. Thanks for the input!
